CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   and nix::Error::UnsupportedOperation}`
   ([#614](https://github.com/nix-rust/nix/pull/614))
 - Added `cfmakeraw`, `cfsetspeed`, and `tcgetsid`. ([#527](https://github.com/nix-rust/nix/pull/527))
+- On Linux, added support for receiving `PTRACE_O_TRACESYSGOOD`
```

Comment:
Let's condense this a bit and add Android: "On Linux & Android added support for receiving `PTRACE_O_TRACESYSGOOD` events from `wait`/`waitpid` using `WaitStatus::PtraceSyscall`"

The absolute path isn't necessary as the function signature makes it clear what enum we're referring to as well.